### ISSUE TITLE
Ability to bulk edit Recipients for multiple/all Aliases

### ISSUE THREAD

**Issue by HTEredrock** [anonaddy/anonaddy#267]
It would be amazing to be able to bulk update Recipients for a selection of Aliases. It would facilitate switching email providers much more easily.

**Comment by willbrowningme** [comment]
Good idea, will try to implement "bulk actions" for aliases when I redesign the user interface shortly.

**Comment by HTEredrock** [comment]
Awesome, keen to see. I got around my initial issue by just changing the default mail, but it would be nice to bulk select if you have lots of mails and multiple mailboxes you need things sent to.

**Comment by willbrowningme** [comment]
This is now possible in the web app.

**Comment by redrockvp** [comment]
Thanks so much @willbrowningme for all the hard work. Legitimately use Addy every day :)
